modules/ggipc-client/src/client.c

```diff
@@ -272,6 +302,12 @@ GglError ggipc_call(
         }
     }
 
+    if (common_headers.message_type == EVENTSTREAM_APPLICATION_ERROR) {
+        *remote_err = GGL_ERR_FAILURE;
```

Comment:
Should return GGL_ERR_REMOTE when there is a remote error

Reply:
Though here we need to translate instead of having the wrapper propagate the error code